examples/flagembedding.py

```diff
@@ -0,0 +1,6 @@
+from FlagEmbedding import FlagReranker
+
+reranker = FlagReranker('BAAI/bge-reranker-base', use_fp16=True)
+score = reranker.compute_score([("hello world", "nice to meet you"),
+                                ("head north", "head south")])
+print(score)
```

Comment:
I don't quite see the value of this example in vLLM. Are you planning to update it to use vLLM components?

Reply:
In RAG scenario, user usually use FlagEmbedding package, I left it as an example, but could be removed.